XenAdmin/Wizards/HAWizard_Pages/AssignPriorities.cs

```diff
@@ -72,15 +72,16 @@ public new IXenConnection Connection
             }
         }
 
-        internal void PopulatePageControls()
+        public override void PopulatePage()
         {
             Debug.Assert(connection != null, "Connection is null; set it to non-null before calling this method.");
-
             UpdateMenuItems();
             PopulateVMs();
             haNtolIndicator.Connection = Connection;
             haNtolIndicator.Settings = getCurrentSettings();
             StartNtolUpdate();
+            IsDirty = false;

```

Comment:
You don't need this either, it won't be taken account of anyway since the page does not override ImplementsIsDirty() any more.